src/cow-react/modules/limitOrders/pure/Orders/index.cosmos.tsx

```diff
@@ -24,5 +24,11 @@ const balancesAndAllowances: BalancesAndAllowances = {
 }
 
 export default (
-  <Orders orders={ordersMock} tabs={tabs} isWalletConnected={true} balancesAndAllowances={balancesAndAllowances} />
+  <Orders
+    orders={ordersMock}
+    tabs={tabs}
+    isWalletConnected={true}
+    balancesAndAllowances={balancesAndAllowances}
+    showOrderCancelationModal={() => void 0}
```

Comment:
nitpick, but should we just add a console log, just in case we want to try out the button works?

Reply:
Thanks @anxolin ! Added